hub_sdk/base/api_client.py

```diff
@@ -104,7 +104,7 @@ def post(self, endpoint: str, data: dict = None, json=None, files=None):
         Returns:
             requests.Response: The response object from the HTTP POST request.
         """
-        return self._make_request("POST", endpoint, data=data, json=json, files=files)
+        return self._make_request("POST", endpoint, data=data, json=json, files=files, stream=stream)
```

Comment:
@hassaanfarooq01 Why are you setting stream here? This is an upload not a download.